index.html

```diff
@@ -20,6 +20,32 @@
         </style>
     </head>
     <body>
+        <article>
+            <h2>Cosmogony metrics</h2>
+            <p>You can check below the number of zones by country for each type of zone, and compare this number with the actual number of zones in the real world.
+                <details>
+                    <ul>
+                        <li>✅✅ : ok
+                        <li>❎❎ : ko, too few or too many zones
```

Comment:
isn't there a red cross ? it's a bit difficult to see the difference with the ticks.

maybe ❌?